src/sentry/models/organization.py

```diff
@@ -283,11 +283,22 @@ def get_audit_log_data(self):
         }
 
     def get_owners(self) -> Sequence[APIUser]:
+        # get owner teams
+        owner_teams = self.get_teams_with_org_role(role=roles.get_top_dog().id)
+
+        # get owners from owner teams
+        owner_team_members = list(
+            OrganizationMemberTeam.objects.filter(
+                team_id__in=owner_teams,
+            ).values_list("organizationmember_id", flat=True)
+        )
 
-        owner_memberships = OrganizationMember.objects.filter(
-            role=roles.get_top_dog().id, organization=self
+        # get owners from org role
+        owners = self.member_set.filter(
+            Q(role=roles.get_top_dog().id) | Q(id__in=owner_team_members)
```

Comment:
it looks like the `organization=self` filter on this query was removed. is this intentional?

Reply:
yes, `self.member_set` is the set of all members of the organization. it's also used here https://github.com/getsentry/sentry/blob/master/src/sentry/models/organization.py#L269

Reply:
ah got it 👍🏼